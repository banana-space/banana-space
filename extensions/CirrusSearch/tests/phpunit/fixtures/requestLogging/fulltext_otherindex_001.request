{
	"type": "fulltext",
	"term": "svg",
	"namespaces": [6],
	"showSuggestion": true
}
