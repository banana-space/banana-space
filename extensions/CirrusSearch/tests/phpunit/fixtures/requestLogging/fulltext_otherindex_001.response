[
    {
        "responses": [
            {
                "took": 6,
                "timed_out": false,
                "_shards": {
                    "total": 5,
                    "successful": 5,
                    "failed": 0
                },
                "hits": {
                    "total": 2,
                    "max_score": 2.7669134,
                    "hits": [
                        {
                            "_index": "commonswiki_file_first",
                            "_type": "page",
                            "_id": "26",
                            "_score": 0.83292395,
                            "_source": {
                                "timestamp": "2016-09-27T20:54:10Z",
                                "title": "OnCommons.svg",
                                "text_bytes": 40,
                                "redirect": [],
                                "namespace": 6
                            },
                            "fields": {
                                "text.word_count": [
                                    7
                                ]
                            },
                            "highlight": {
                                "text": [
                                    "File stored on commons for test purposes"
                                ],
                                "title": [
                                    "OnCommons.<span class=\"searchmatch\">svg<\/span>"
                                ]
                            }
                        },
                        {
                            "_index": "commonswiki_file_first",
                            "_type": "page",
                            "_id": "27",
                            "_score": 0.8020882,
                            "_source": {
                                "timestamp": "2016-09-27T20:54:14Z",
                                "title": "DuplicatedLocally.svg",
                                "text_bytes": 45,
                                "redirect": [],
                                "namespace": 6
                            },
                            "fields": {
                                "text.word_count": [
                                    7
                                ]
                            },
                            "highlight": {
                                "text": [
                                    "File stored on commons and duplicated locally"
                                ],
                                "title": [
                                    "DuplicatedLocally.<span class=\"searchmatch\">svg<\/span>"
                                ]
                            }
                        }
                    ]
                },
                "suggest": {
                    "suggest": [
                        {
                            "text": "svg",
                            "offset": 0,
                            "length": 3,
                            "options": []
                        }
                    ]
                }
            }
        ]
    }
]
