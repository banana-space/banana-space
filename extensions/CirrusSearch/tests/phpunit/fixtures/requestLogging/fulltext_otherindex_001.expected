[
    {
        "channel": "cirrussearch-request",
        "context": {
            "$schema": "\/mediawiki\/cirrussearch\/request\/0.0.1",
            "all_elasticsearch_requests_cached": false,
            "elasticsearch_requests": [
                {
                    "cached": false,
                    "hits": [
                        {
                            "index": "commonswiki_file_first",
                            "page_id": 26,
                            "page_title": "File:OnCommons.svg",
                            "score": 0.83292395
                        },
                        {
                            "index": "commonswiki_file_first",
                            "page_id": 27,
                            "page_title": "File:DuplicatedLocally.svg",
                            "score": 0.8020882
                        }
                    ],
                    "hits_offset": 0,
                    "hits_returned": 2,
                    "hits_total": 2,
                    "indices": [
                        "wiki_general",
                        "commonswiki_file"
                    ],
                    "limit": 21,
                    "max_score": 2.7669134,
                    "query": "svg",
                    "query_type": "full_text",
                    "search_time_ms": 6,
                    "suggestion_requested": true,
                    "syntax": [
                        "full_text",
                        "full_text_querystring",
                        "simple_bag_of_words"
                    ]
                }
            ],
            "hits": [
                {
                    "index": "commonswiki_file_first",
                    "page_id": 26,
                    "page_title": "File:OnCommons.svg",
                    "score": 0.83292395
                },
                {
                    "index": "commonswiki_file_first",
                    "page_id": 27,
                    "page_title": "File:DuplicatedLocally.svg",
                    "score": 0.8020882
                }
            ],
            "http": {
                "client_ip": "127.0.0.1",
                "has_cookies": false,
                "method": "GET"
            },
            "source": "cli"
        },
        "level": "debug",
        "message": ""
    },
    {
        "channel": "CirrusSearchRequests",
        "context": {
            "elasticTookMs": 0,
            "executor": "123456789",
            "hitsOffset": 0,
            "hitsReturned": 2,
            "hitsTotal": 2,
            "identity": "2c33c9c64851a02e2ba8143c6550a3b8",
            "index": "wiki_general,commonswiki_file",
            "limit": 21,
            "maxScore": 2.7669134,
            "query": "svg",
            "queryType": "full_text",
            "source": "cli",
            "suggestionRequested": true,
            "syntax": [
                "full_text",
                "full_text_querystring",
                "simple_bag_of_words"
            ],
            "tookMs": 0
        },
        "level": "debug",
        "message": "{queryType} search for '{query}' against {index} took {tookMs} millis and {elasticTookMs} Elasticsearch millis. Found {hitsTotal} total results and returned {hitsReturned} of them starting at {hitsOffset}. Requested via {source} for {identity} by executor {executor}"
    }
]